packages/core/package.json

```
@@ -6,11 +6,16 @@
   "homepage": "https://platform.zapier.com/",
   "author": "Zapier Engineering <contact@zapier.com>",
   "license": "SEE LICENSE IN LICENSE",
-  "main": "index.js",
   "types": "types/index.d.ts",
+  "exports": {
+    "require": "./index.js",
+    "import": "./index.mjs",
+    "types": "./types/index.d.ts"
+  },
```

To fix the failing tests due to [the imports](https://github.com/zapier/zapier-platform/blob/9019b6f403587fc1e5ec97c9f9730a1f26c7ad8f/packages/legacy-scripting-runner/test/integration-test.js#L13-L16) in legacy-scripting-runner, maybe we need to do this:

```suggestion
  "exports": {
    ".": {
      "require": "./index.js",
      "import": "./index.mjs",
      "types": "./types/index.d.ts"
    },
    "./src/*": {
      "require": "./src/*.js"
    }
  },
```